shiny/ui/_input_numeric.py

```diff
@@ -39,6 +40,11 @@ def input_numeric(
         Interval to use when stepping between min and max.
     width
         The CSS width, e.g. '400px', or '100%'
+    update_on
+        When should the input value be updated? Options are `"change"` (default) and
+        `"blur"`. Use `"change"` to update the input immediately whenever the value
+        changes. Use `"blur"`to delay the input update until the input loses focus (the
+        user moves away from the input), or when Enter is pressed.
```

Comment:
Would be great to surface this addition (as well as anything you think is worth mentioning) in the CHANGELOG